pubsub/google/cloud/pubsub/message.py

```diff
@@ -85,7 +86,13 @@ def from_api_repr(cls, api_repr):
         :rtype: :class:`Message`
         :returns: The message created from the response.
         """
-        data = base64.b64decode(api_repr.get('data', b''))
+        try:
+            raw_data = api_repr.get('data', b'')
+            padded_data = raw_data + b'=' * (4 - len(raw_data) % 4)
+            data = base64.b64decode(padded_data)
+        except (binascii.Error, TypeError):
+            data = api_repr.get('data', b'')
+
```

Comment:
Have we verified that the totally-unencoded bit actually happens (as opposed to the stripped padding)?